xo_install.sh

```diff
@@ -1,24 +1,46 @@
 #!/bin/bash
-sudo apt-get install --yes nfs-common
+
+xo_branch="stable"
+xo_server="https://github.com/vatesfr/xo-server"
+xo_web="https://github.com/vatesfr/xo-web"
+n_repo="https://raw.githubusercontent.com/visionmedia/n/master/bin/n"
+yarn_repo="deb https://dl.yarnpkg.com/debian/ stable main"
+node_source="https://deb.nodesource.com/setup_5.x"
+yarn_gpg="https://dl.yarnpkg.com/debian/pubkey.gpg"
+n_location="/usr/local/bin/n"
+xo_server_dir="/opt/xo-server"
+xo_web_dir="/opt/xo-web"
+systemd_service_dir="/lib/systemd/system"
+xo_service="xo-server.service"
+
+#Install node and yarn
 cd /opt
-curl -sL https://deb.nodesource.com/setup_5.x | sudo -E bash -
-curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
-echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
-sudo apt-get update
-sudo apt-get install --yes nodejs yarn
-curl -o /usr/local/bin/n https://raw.githubusercontent.com/visionmedia/n/master/bin/n
-sudo chmod +x /usr/local/bin/n
-sudo n stable
-sudo apt-get install --yes build-essential redis-server libpng-dev git python-minimal libvhdi-utils
-git clone -b stable https://github.com/vatesfr/xo-server
-git clone -b stable https://github.com/vatesfr/xo-web
-cd xo-server
-sudo npm install && npm run build
+/usr/bin/curl -sL $node_source | sudo -E bash -
+/usr/bin/curl -sS $yarn_gpg | sudo apt-key add -
+echo "$yarn_repo" | sudo tee /etc/apt/sources.list.d/yarn.list
+sudo /usr/bin/apt-get update
+sudo /usr/bin/apt-get install --yes nodejs yarn
+
+#Install n
+/usr/bin/curl -o $n_location $n_repo
+sudo /bin/chmod +x $n_location
+sudo /usr/local/bin/n stable
+
+#Install XO dependencies
+sudo /usr/bin/apt-get install --yes build-essential redis-server libpng-dev git python-minimal libvhdi-utils nfs-common
+
+/usr/bin/git clone -b $xo_branch $xo_server
+/usr/bin/git clone -b $xo_branch $xo_web
+cd $xo_server_dir
+sudo /usr/bin/npm install && /usr/bin/npm run build
 sudo cp sample.config.yaml .xo-server.yaml
-sudo sed -i /mounts/a\\"    '/': '/opt/xo-web/dist'" .xo-server.yaml
-cd /opt/xo-web
-yarn install --force
-cat > /etc/systemd/system/xo-server.service <<EOF
+sudo sed -i "s|#'/': '/path/to/xo-web/dist/'|'/': '/opt/xo-web/dist'|" .xo-server.yaml
+cd $xo_web_dir
+/usr/bin/yarn install --force
+
+if [[ ! -e $xo_systemd_service ]] ; then
```

Comment:
@hooksie1 we're calling xo-server.service you're naming it xo_service and it fails to execute. 

once the build is complete.